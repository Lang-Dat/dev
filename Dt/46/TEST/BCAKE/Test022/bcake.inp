2379
1025 4177
2307 1634
4236 4919
2032 1000
2511 2050
1616 4438
3618 2615
2513 4244
3354 2745
2475 4393
3533 4192
2208 1830
1933 3657
2871 1929
3395 4747
4161 4376
4304 1219
2683 3485
3748 1672
1253 4339
3588 3738
2936 2799
1778 1465
2183 2773
1569 4293
4024 3750
2721 3195
1961 3867
1334 4169
1304 1299
3749 2861
2673 3803
2055 3883
1492 3716
3443 4230
1537 4544
3772 1374
1582 2644
2184 3226
4620 2964
1869 2960
3815 3665
3132 1913
2925 2206
1675 4004
2935 2386
4771 1437
1688 1419
4851 1319
1699 4644
1771 1476
1587 2245
3022 2163
4069 4047
4629 3892
1000 4387
1877 2948
1618 1105
3048 2232
1361 2637
2990 2757
3425 1160
1608 3182
3325 2743
2708 2883
3007 2327
4871 2006
2725 2759
3362 4287
2754 1081
3098 2104
3451 1517
1724 3196
2061 1910
4844 2362
4636 3436
2823 2841
3440 1313
1320 4693
2716 1428
2565 4241
3168 4006
4148 1952
2242 1789
1409 1886
2299 1646
2244 4209
4602 2746
1469 4262
4181 1883
3364 1840
4385 3806
1240 1845
2377 2618
3185 1133
1418 3980
4192 4402
2014 4520
2741 2135
3802 1917
4175 3961
2473 2731
1845 4622
3292 2685
2600 2110
4174 1716
1480 1391
1497 1565
4562 2442
3940 1802
3995 2057
3590 4845
2988 1279
3640 4597
1843 4738
4163 3656
3456 1774
3238 4603
3809 4691
3285 1007
3266 2063
4386 2759
3785 1441
2455 3986
4542 4759
2771 1900
2091 1251
3977 3031
4240 2901
4214 2212
2071 1325
3327 3775
3220 1937
3697 4679
4295 3023
2073 1634
4385 3556
4351 2802
1751 2124
2723 1438
4230 4434
3395 2774
4719 2380
1154 1345
4178 1667
3200 1868
4585 3636
3071 3093
1381 1572
1702 3417
4598 3692
1297 1499
3515 1857
1321 4318
2577 3561
4053 1567
1070 4707
3170 2015
1285 2997
3118 1438
1447 1438
2613 3325
2579 1911
3660 2936
4153 3320
1450 4814
3558 3976
1463 2777
3858 2122
2871 4426
3920 2259
2669 2015
3067 1898
4041 3360
1714 4950
3422 4231
4140 1756
2819 2649
1954 1668
3905 2472
3799 4273
2902 4179
2755 1596
3487 2722
3227 3826
1356 4090
1044 3693
4629 1825
3902 2761
4913 2765
2497 1129
1701 2650
4856 3340
2323 3673
2589 2053
3106 2192
1898 1120
3617 4651
3046 4447
1285 4343
4070 2530
4085 3830
4231 1584
4038 3376
1256 4570
3239 2345
1353 4161
4529 1488
3919 4368
3142 2826
4871 1138
2846 2991
2883 4772
3981 2254
1331 4040
1962 1385
3871 4549
4906 1002
4128 2212
4666 3058
2177 2274
4972 4946
4922 4456
4635 3678
4596 2662
3190 4202
1396 2198
2466 2519
4434 3091
2212 2940
1367 1023
1717 2841
2496 4621
1399 1457
1982 2331
1784 2562
4306 2116
1314 2101
3780 3105
1054 3670
2069 2154
1543 1582
1911 4307
1005 3286
4800 1331
3396 2015
2097 2412
3091 4975
2193 2022
3317 1751
2688 1051
2295 4031
1860 2143
1615 2732
3512 3973
1885 2589
2774 1249
3911 4493
3379 3159
1104 4066
2560 3956
3437 1589
2924 2314
1917 3617
1501 4420
3717 2247
1782 1876
4385 4351
1943 4441
2805 2011
4647 4810
3788 3035
2082 4776
3058 3669
4898 3220
4745 3864
2299 2234
2955 2326
3979 3801
4477 4552
2139 3411
1670 4534
4581 3175
2595 1510
1384 1923
2330 4267
2722 3528
1107 2835
1283 1563
3380 4962
1711 2736
3561 2248
2339 2013
1586 3655
4068 1984
1536 4799
1662 2068
2396 2456
1925 1384
2341 4365
3449 1075
1745 3681
1201 1253
4088 4259
4107 4355
2985 2355
2735 2528
1121 3387
4951 1281
3528 2372
1438 2310
3848 4558
4164 1811
3152 3350
4360 1025
2798 3157
4004 4767
3631 4281
1305 1388
2382 1624
2424 3546
2239 3742
1829 2563
1622 1555
4876 1019
4416 2178
1445 3821
4663 3262
3355 3434
4665 4970
3454 3454
2836 2647
2750 4405
4567 3190
1218 4262
3887 1373
1876 2431
1931 4202
3113 3601
3970 4121
4369 4734
2824 2872
2563 2018
1958 2500
2685 3467
3189 4893
1320 1923
3152 3925
3820 4875
2119 3253
3780 3685
4701 1420
3631 4811
4604 4917
2461 1570
3701 4378
2561 3926
4337 1275
3398 1595
1421 2495
3333 3717
2400 1803
4371 1226
2091 3156
4729 1549
1700 3437
2902 2041
4293 1135
4637 3905
2523 3341
1088 2287
2501 3605
1631 1901
1679 1916
1047 1063
3415 4939
2752 3490
3281 2273
4238 4833
4575 2051
2334 3616
1988 2120
4933 4386
1338 2929
3703 1698
1470 2088
3957 4137
3005 3854
4125 4541
1199 4828
2906 1340
1586 1857
3886 3450
3919 1607
3597 4340
1011 1669
1029 2445
3839 1092
2264 1357
2415 1218
2449 2751
3255 1546
3025 1884
3763 3634
2043 3841
3520 4486
4484 3123
1391 4556
4911 3460
2386 4596
2317 3981
3603 1515
1116 3560
3613 4445
1807 2147
4445 4490
1035 2198
1870 3203
2920 3662
2029 3178
3082 1156
4872 1649
1109 2242
2229 4797
4770 4411
1801 2541
3766 2329
4908 4195
4769 4765
1379 3914
4882 4274
3126 1803
3922 4732
1110 2399
1631 4514
3619 3041
4894 1442
1185 4742
2902 1806
1922 3775
4418 4581
4593 1760
4998 3947
2076 1772
4617 1041
3708 4353
3815 4364
2131 3834
3740 1625
2490 3228
3478 1593
2478 3554
3940 2188
2745 4215
3776 4057
1847 4825
1040 2277
4754 2447
3006 4137
2221 1153
4690 1644
4384 4410
1609 4728
4593 1951
2632 1041
1751 1060
1091 3291
1513 2700
4680 3455
1046 1615
2704 2814
4952 3113
4852 4715
3332 3506
4796 4584
2588 1546
3791 1146
3208 2361
1443 1297
1687 4877
3771 3075
3547 4092
3887 1308
3407 3285
1326 1871
2302 3025
4305 3528
3001 2575
4349 4016
2713 3583
1997 1910
3613 1296
4206 2356
3880 4684
4002 2337
3194 4479
4578 4788
4367 2105
1653 1797
4545 2661
2709 1135
3210 1233
2045 3053
4590 1162
2040 2001
2822 2475
3570 2497
4760 3308
2059 3555
3361 2812
4831 4831
2206 3701
1640 1408
1675 3364
1488 1843
2781 4332
2372 1374
3400 3315
4178 3115
3563 4358
1955 4254
2242 3786
1747 4097
3351 3578
2084 1695
4054 4284
2593 2338
3940 4950
4480 1205
3723 1979
2934 1473
1782 1615
2729 2724
4690 2392
1281 2049
3819 3299
2178 1017
3939 3478
2699 2738
1629 4258
3743 1307
4905 4238
3177 4538
3705 1644
1396 2656
4965 4864
1389 2775
2111 1917
4503 3527
2446 2894
4958 2802
3738 1385
4652 4557
1305 3567
1673 4371
4160 2928
4983 4871
2240 4225
3969 3130
1296 4301
3057 4098
3009 3910
3170 1378
4092 1500
4824 2192
2589 4644
3106 4312
4610 2887
2239 3595
4518 3059
4976 4162
2308 3719
2997 4641
1147 4106
1463 4698
4249 3492
4277 3461
2750 2791
2053 4986
4513 2938
3845 4309
3625 2195
4319 2939
3421 2908
4425 3949
1507 4632
2987 2771
3082 2134
2047 2605
3083 4253
1641 2654
4628 1760
2905 1589
3627 2690
3170 3048
4406 4612
1934 1904
2611 1230
2341 2584
3012 3986
2772 2711
4441 4346
1722 4996
4661 1839
1100 4082
1228 1382
4662 4534
4843 3138
3133 1231
2856 3541
3762 3499
3531 3640
4311 3143
2920 4524
3329 3774
2955 4336
1283 2190
3628 3423
1902 2860
4795 3333
4296 2124
1705 1080
3552 3326
3806 3585
3244 3239
3412 1141
4794 4282
3869 3927
4329 3662
3822 1719
3473 2142
1356 1644
3791 2815
3464 1731
1547 3308
4342 4393
4318 4536
1365 3843
3567 3795
1508 2082
4384 1963
4631 1411
1039 2592
3142 4141
2775 1534
1440 3334
3164 4890
1891 1592
4372 1405
4407 1018
1181 1544
3564 1151
3550 3014
3150 2857
4638 4678
3176 3153
3160 2259
1788 2591
2580 1496
2174 4647
3163 2223
2887 4543
3931 4120
3626 2300
4980 2625
1930 2109
1727 4177
1267 2994
4596 3975
1342 3777
4973 4121
2194 3556
3724 2470
2178 1246
2074 3211
3735 2868
1842 4362
1813 3982
2425 4852
3323 3425
2901 2971
4391 3911
1066 1320
4528 1558
3593 1257
1154 2593
4938 2588
1203 4843
1255 4483
4816 4144
1068 2609
3790 4548
1888 3388
3403 3533
3541 4076
3622 4740
1829 1607
1044 2322
2143 1500
3388 1066
3279 2048
4380 4699
1159 2587
4868 1392
3382 3265
2393 4910
3538 1737
2013 4365
4361 3204
3061 4254
3204 4203
2042 2604
4749 1776
3726 1539
3330 4749
2314 2945
2458 3586
4861 3607
1865 1603
1443 3117
2565 3444
1576 4000
2462 3700
4012 3211
1979 4137
3230 1784
3669 1930
2615 3661
1797 1422
1194 3410
4795 4187
2986 2956
4972 2409
1536 1438
3947 1915
2105 3871
2971 1980
1910 2471
1683 3242
4438 4424
2548 1525
3448 3055
2979 2734
4448 4767
3356 3257
1643 3771
4749 4143
3059 3248
4498 1746
1686 4566
4224 2104
1032 4041
3374 3366
4910 2073
1455 4323
4542 3832
2030 2668
2289 4034
4932 2683
1422 2749
4198 1164
3614 1261
4478 1741
2488 3360
4310 1936
2949 3498
3558 3918
4900 1088
4897 4487
2901 2180
3069 3181
3597 1467
1118 4262
1273 4887
1888 4456
1762 3851
1844 4100
3272 1931
1832 3522
3865 4272
1292 2534
4869 1510
2875 2622
1310 1616
1422 3924
4219 3476
1496 3076
3016 3409
2949 2035
3640 3397
3439 3483
2068 4106
3368 2406
4785 1675
3060 2678
2812 2619
2390 3944
1552 4517
4984 2779
3186 2351
3323 4456
2381 4015
4639 3619
3596 2456
4317 4190
1978 3527
3563 3203
3094 1655
4401 4479
3314 1980
2076 3201
4886 2595
2714 4281
1171 3852
4451 4691
1699 1313
3034 1376
2038 2340
4754 3744
4508 1739
1846 1429
4224 4977
4534 3613
2029 1326
3288 2085
2671 2354
4529 2480
3104 4027
3905 4395
4212 3450
1052 2066
4550 4048
2067 2724
1574 4164
3665 3350
2177 4763
1122 4140
4119 1465
4871 3856
3332 2660
1253 3717
4830 3218
2580 4678
4379 4113
1764 4224
2149 1853
3046 3106
3804 4734
2006 4000
3437 1452
1229 1677
4967 3661
4090 1566
1209 2180
3044 2898
3137 1811
2424 4948
4433 2122
4229 2721
1170 2313
4926 4092
2119 4217
4616 4565
4747 2911
4647 3381
4544 2110
2181 4508
2459 3083
4206 4157
3121 2855
4148 4012
4678 1457
1410 4744
1644 3105
3495 1263
3393 4712
4684 3737
3621 2605
4136 1814
2348 4991
1444 1722
3540 4604
3138 2748
3950 2903
3384 1465
4602 4709
3742 2736
3989 3750
4026 4297
1886 4969
1031 3253
1921 2416
3356 2491
3181 2070
1700 3150
1638 4850
1161 3859
4225 4316
3310 3935
1260 1531
4996 2643
3299 1592
3256 2403
2650 4250
4257 4330
1718 4375
2819 3092
2208 2461
1596 4290
1598 3082
2803 4470
3534 3578
2939 3564
2826 1365
3616 2510
4111 2434
3911 1109
3473 3785
4798 4271
4601 2865
4813 1707
4241 1517
1321 1475
1106 1377
3839 1853
2005 4641
3336 4321
2883 1476
3931 2754
2420 3132
3897 1647
4703 3429
4562 3942
2978 1467
4630 4565
4211 2427
1321 4618
2679 3535
2986 2205
3744 4653
4129 1698
4237 3864
2543 1770
3377 2373
3138 3316
3498 4707
3017 3525
1247 1668
2313 3823
1973 3358
4240 2624
2607 3103
2976 1459
3808 4613
1127 1975
3009 2919
4874 2137
1179 1583
3294 2704
3634 4054
1028 2271
3310 1709
2167 1396
4217 1253
3020 2016
1781 2521
1509 4628
4347 2216
1937 4788
4683 3122
4441 3420
3197 3655
3468 2235
4604 2223
3437 1563
3579 2347
1766 1098
2953 2090
4691 1908
2968 1324
2214 3540
1176 4870
2516 1752
4177 1245
3164 3232
4867 3922
1681 3373
2849 4408
1420 4210
3301 4098
1977 1194
3642 3141
3027 1854
3182 1128
4460 4952
1666 1672
4060 3831
1281 4384
1011 2220
3875 2406
4999 2446
4961 4665
1313 4771
2794 2171
2836 2106
1133 3632
4374 4101
1941 1204
3197 4402
1579 3147
4591 2534
2519 2235
2274 4047
1464 1985
2388 1068
3406 2464
1291 3648
2216 2537
3796 2383
1937 2164
3938 3399
2204 4129
3314 4895
2466 3678
4318 1626
3105 4579
4607 4461
4039 3575
1481 1901
4765 1446
2338 3262
1152 3026
1829 3792
2615 3092
3044 3820
4572 3291
3934 2557
1906 1158
4613 2750
4992 3319
3515 1997
3126 1744
4913 2311
4176 3169
4258 4962
4902 3422
3838 3736
4609 2644
4653 3474
4102 1765
1469 2988
1874 4894
4278 1832
3655 2922
3876 2986
1311 2908
3227 4514
1969 1394
2391 4473
3027 1849
1315 1757
1603 1794
2953 2526
1584 1837
2920 4776
3902 3382
4327 3762
3619 1014
2582 4297
2843 4714
3111 4905
1506 3196
1182 2864
2945 2991
2865 2650
2572 1924
1510 3920
1950 4559
3943 2429
2579 2491
3938 2898
3826 1855
3415 1009
2141 3590
2688 3299
4168 1876
2589 2553
1710 4517
4574 3454
4594 1878
2985 4304
3734 4913
2027 2459
3814 1275
2078 1308
1618 2688
3894 2040
2589 2672
2069 3456
2697 1671
3465 3405
1808 4488
3260 3073
4598 1673
3590 1492
1420 3500
2794 1346
4862 2918
4819 1603
2675 1256
4897 3582
2264 3153
2366 1936
1111 1367
2372 2672
2967 4020
4708 3355
1777 3064
1027 1034
3058 3395
1822 1740
3514 1282
1453 2721
1750 2042
3269 4356
1233 4966
4758 3766
3954 2855
3913 2679
4354 1972
2701 3946
2283 3063
1497 4367
4175 1496
3121 4079
2409 1345
2386 4320
4717 2128
3463 3275
4846 1300
1707 1825
4052 2583
1364 3673
3935 4940
1020 3741
3382 1738
3585 3140
2571 4286
3081 2009
3543 3388
1815 2020
3053 4053
1039 2723
3016 1709
2665 4895
4759 4156
3171 1313
1545 1410
3374 2173
3242 2061
2057 4892
4212 4270
1547 1277
1241 3951
4567 3128
1958 1961
2683 2820
3824 1537
2746 4716
3198 2134
1279 3634
2478 4165
4127 2155
3128 2806
2580 1487
1535 1921
3914 4137
4927 4342
4523 4770
1470 3079
4895 4773
4981 1283
1697 2515
1284 1026
1651 3746
3378 2316
3862 3962
4349 1190
1226 3300
5000 1110
1041 3395
2550 4084
2547 2442
1556 2911
1102 1417
3651 2136
3766 3936
2781 4551
2389 4899
3396 4181
2807 4048
4946 2409
1883 4651
3423 2868
2338 1032
2789 1938
3315 2638
2049 3096
1006 2541
2597 3608
4610 1863
4443 3592
1708 2435
2140 4958
3601 2502
3615 1997
2511 4030
3003 1398
4665 4805
4997 1064
1942 1797
4799 2087
3773 3112
3242 2066
2265 3645
2356 2070
4722 2536
2221 4475
2550 2183
2757 2070
1793 3488
1939 3804
2249 4384
4356 2175
2679 4886
4994 2387
2778 4608
4147 3011
1656 2874
1201 2277
1748 1748
3515 1799
2665 1480
3370 3129
2391 2454
3571 1943
4271 1076
2397 4404
2584 2070
3289 2276
1023 2743
2158 3696
3486 4301
1854 3182
3478 1481
2929 1184
3861 1909
3419 4743
4620 2528
3486 2493
4767 1717
4250 4065
2102 4853
2377 3533
4432 1483
3003 1882
1866 3761
2804 1414
4793 1437
1788 3719
2917 2209
4029 1362
3475 2867
1013 2042
1322 3565
2766 4122
3430 1138
2936 3748
1571 1556
1945 2377
4194 4302
1379 1300
2341 4303
4643 3973
4779 1602
2166 2711
2221 3867
1854 3071
4053 4192
3106 3310
3654 2604
3025 3346
1216 3617
3706 1515
2376 2560
2744 4538
2226 1632
1128 2007
1049 4135
2655 2013
3414 1321
1690 3028
1853 2910
1263 3935
1134 2671
2785 3491
3571 2287
4301 2616
2122 4493
4263 2562
4382 1448
2700 2382
3924 2534
2218 1959
3909 3885
4040 3700
3464 2475
1256 3841
1943 3908
1322 4874
2030 2703
3915 4729
4849 4269
4830 4209
2649 1281
2891 2349
1074 3344
3722 2280
1970 2675
1762 3428
4240 3781
4667 2385
1103 2421
2115 2566
4334 4996
2699 3699
1733 2963
2204 2394
1564 3882
3921 4285
4147 4628
1740 1241
2564 1598
3824 3144
1821 2740
3966 2778
2239 4413
1619 1504
2335 1055
1884 2383
2224 4991
2147 2398
3754 4838
2792 1046
1351 1386
3784 1919
1552 2833
3379 1683
1450 4507
4130 4938
1193 4952
1716 1725
2929 2314
4717 4236
3462 2840
2416 2396
3009 3357
3202 3943
4355 2937
2086 2077
4410 4025
4621 1806
2869 1933
1005 2450
4010 3404
2751 3459
4217 4773
2707 4404
3664 2930
3181 2038
4210 3640
4134 4545
2127 3034
3515 1193
3134 2909
4375 2304
1274 3860
3320 4994
4982 4415
2212 2877
4257 3382
4979 3432
4314 2264
2683 1882
4936 1005
2070 4500
4719 3850
1815 4933
3970 2171
2791 2639
1280 4488
3315 3047
2662 3964
1628 4317
2833 2401
1119 4550
4501 2190
4065 4726
2602 2359
1536 2787
4132 3218
4882 4246
1874 3830
2863 2501
2855 3742
4985 4080
1140 1390
2071 2393
4666 2812
2223 2737
3333 2409
3934 1315
2329 1877
3709 3274
2054 3595
3069 2652
1411 1201
4967 1740
2948 3150
3901 3954
4695 1693
3060 4491
2013 2775
4618 4699
4839 3813
3314 4297
1383 4661
3549 1111
1541 4028
3020 4259
2771 3128
2218 1513
1475 1031
4846 1114
4233 2999
4168 4865
3796 3525
3657 2776
3144 4310
4565 4013
4765 3022
4278 4259
1021 1191
1156 3466
3952 3776
3641 2921
4455 1583
1466 2238
4877 1359
1168 2590
3250 1521
3301 1226
3540 2168
4563 2510
2641 4134
3366 4598
2094 4741
2829 1928
2334 4864
3821 2560
1692 4296
3861 1142
4120 3324
3250 1293
1369 3562
1900 4918
1451 2184
1458 1666
2129 2041
3358 2990
1293 2117
4886 3179
2383 4669
3393 3192
4372 3598
1267 3455
2562 4003
2080 2062
1407 4441
4237 4655
3345 3908
1954 4721
3893 4066
3339 1029
1485 4271
2484 1868
4596 1827
1191 1243
1127 1647
2824 4342
2797 3716
2477 2277
3680 2954
4983 4227
4148 1802
3228 2608
1113 2261
3009 2256
2738 2566
2387 1062
2990 1947
2906 2419
3942 1977
3721 2187
3782 2186
1723 2728
1111 4581
3920 3486
3665 4424
2814 3328
4805 1912
1677 3674
4597 1330
1097 4869
3044 2341
1418 3610
4165 4881
1970 3347
4359 1503
3143 2085
3737 1089
1936 3478
1495 2890
1124 3146
3595 2822
1350 3069
2047 4904
3587 1968
4000 4783
1705 1868
1862 2340
2902 1539
1703 1703
1793 1013
4952 3346
4619 1884
4917 1834
4353 4437
1434 1645
1030 3171
4165 4168
4763 2033
1634 3098
2039 3043
3497 2301
1557 3084
4515 1063
4531 2579
3567 3123
1049 1240
1617 2837
1630 3539
4252 2792
4608 2566
3802 3956
2913 2216
3722 3993
4285 2726
2164 1160
4145 2202
3735 2415
1476 4405
3660 3307
2625 3255
3670 4045
4797 4084
2222 4575
3102 4400
1041 1119
3852 4637
4529 2346
1545 2972
2829 3945
2988 3425
3048 2497
3653 4302
2207 4427
1329 2528
2376 1891
4131 1611
1484 4033
1821 3892
3365 2114
3412 2448
2624 4815
2487 3636
1367 2400
2244 2435
3297 1183
2394 2048
4187 3691
2279 2863
3668 1390
2601 1613
3099 3665
4151 4960
1870 4449
3632 2979
3116 3121
4840 2675
1327 4981
3781 4030
3803 4519
1109 3781
3999 3833
4713 3120
2132 1905
2993 4293
2302 4748
3547 3183
4938 2039
1887 1956
4652 4001
4334 1507
3493 1999
3182 4945
2706 1487
2640 4163
1681 3961
4192 2443
4212 1715
4426 1515
4922 2229
1471 4824
4611 1198
2594 1379
4521 2936
1672 3373
3523 4793
4037 1358
3212 4702
4924 2483
4340 4878
1738 4656
3536 2853
3750 3013
4722 2200
3741 3485
1807 1676
3888 3195
2140 4036
3949 2519
2999 4068
4034 1771
4789 4785
2265 3787
4873 3132
2814 1824
1006 2950
4979 2737
1260 2140
4012 1489
1494 3463
3485 4219
1149 3729
2831 2782
2748 2580
4835 3920
4408 1004
1231 4327
2068 2610
1366 2540
1702 2072
2244 1883
2093 4097
2570 2495
2457 2228
3588 3753
2760 4517
1243 1286
2083 1828
4590 3973
2933 3539
2696 2832
2361 1509
2255 3330
3957 2768
4282 1554
4444 3710
4005 1696
4159 1177
3967 3144
4680 3865
2812 1610
4716 3797
4020 4362
2566 3479
4625 3896
4639 3204
4344 4884
4030 1122
2477 1819
4929 2973
1524 2607
3402 2189
4191 3011
4191 3050
1168 3750
2234 1004
3989 3347
2134 4479
3880 2039
1796 1674
1747 3564
4396 2706
2771 1299
4322 2850
2948 4358
4622 3757
2411 3055
2412 1769
1943 4145
1367 2374
4811 4775
1505 4426
1765 4703
2025 3193
3895 4303
3343 2063
2237 1466
4678 4481
2968 4090
4165 1064
3641 1785
3590 3791
3498 4088
4059 4239
3180 4135
2290 1360
3067 4798
4868 3074
2087 3094
1538 3502
3272 1526
3206 4447
1994 2747
1125 4396
4035 3855
4181 2776
3387 3655
1771 4423
2342 3951
2378 2020
4224 3605
3854 1398
2805 3880
3628 3534
2078 1331
3716 4347
3265 2801
4620 4027
1338 1762
4970 1991
3789 2714
2715 4678
2488 1432
1508 4307
3106 3102
1138 4901
4183 1860
3638 3145
4488 2577
1117 4746
1634 4403
3730 4440
3040 2011
4375 2092
4347 1176
3715 1719
4910 3075
2441 1477
4147 3681
2005 1879
2121 1012
2594 3986
3847 2667
1155 2890
2392 1326
3588 3066
2840 1729
4189 1989
3164 4536
2905 2110
1223 4104
2783 1867
2795 1729
3496 1566
4593 3190
4515 2530
2981 3826
2967 1649
1231 1825
3964 1433
4812 4451
4224 3686
4835 2296
2547 3771
1382 2019
3542 2591
1025 1313
2730 4217
4420 3603
2005 3694
4253 3497
3851 4897
3708 2126
4933 1428
2387 3865
4429 2906
4177 1600
2264 3367
3809 4423
3758 4597
1893 2752
1259 1519
2333 4079
2234 1883
1188 2740
1668 1765
2930 3373
1856 2241
1844 1025
1883 3222
4332 2298
1986 3333
4227 2099
3096 3553
2838 1158
1604 2439
4498 3061
2013 2528
3353 4527
4234 4656
3713 1257
2354 3593
2469 4568
2234 1102
3779 2410
1712 3228
3099 2945
2087 1624
2270 2503
1229 2827
2581 3048
1013 3755
3106 3511
4531 4722
4353 1075
2018 2315
4711 2394
4718 1457
3399 4309
3591 4947
4975 1456
1310 4684
2984 3125
3875 2372
3439 3872
1915 3767
3664 3245
2676 3770
1754 4456
2506 4396
3577 4424
1400 4050
2233 4028
4960 1020
2611 4770
2888 1350
3780 2653
1968 3815
1038 1447
1623 1174
1277 2761
3040 4050
3906 3902
3219 4900
4145 2891
2347 4458
1711 2776
2085 1693
2597 1926
2941 3201
4862 1024
1707 2565
2864 3750
2225 3368
1927 3202
2009 3296
4615 1353
1854 3550
3097 2995
2737 1724
2597 3643
2235 2885
1642 4260
3404 1821
4702 2436
3326 2212
2085 4839
1468 4002
3025 3324
2711 3237
3863 1141
3282 4952
4159 1796
2755 3415
1711 1614
4833 2898
1479 3893
4480 3420
2336 3716
2551 2415
1339 3802
1809 3742
2078 2925
2105 2748
3990 3655
4577 4316
2771 4559
4741 4302
2251 1406
2104 1628
1092 4419
1864 4388
1417 2550
3134 2350
3059 4282
1502 3638
4120 2416
1303 2361
1728 2153
2123 3955
2790 2198
1575 4656
4346 1017
1887 1035
4911 2365
3505 3975
3125 4270
1177 1859
3829 4906
1065 1627
4377 3577
2806 4133
3280 4430
2827 2499
1507 4341
3543 4417
3684 2611
3093 1632
4841 4404
4506 2660
1917 2923
1164 2352
2653 1291
1770 4931
1370 4606
2475 3993
2555 2357
4898 4396
2456 1542
4067 2104
1963 2243
1991 3507
4645 1498
1191 1367
4290 3970
3143 3522
2298 4423
1395 3335
3932 3451
3045 3752
4429 3240
2856 2172
1680 2412
2629 3626
3995 4298
1053 2368
3066 2676
4022 2011
1517 3677
4830 4286
1930 4340
2649 2757
2311 2180
1953 4544
1646 1358
4642 2187
4231 1793
4960 1420
3975 3235
3430 3805
4591 1650
3108 1430
2867 4770
3840 3564
2063 2570
1972 1363
4407 1151
2866 1026
4072 1906
1869 1844
3686 3673
2851 2248
3095 1390
3663 2524
4212 3043
3895 2063
1688 4042
3283 3164
4267 1486
4053 4132
2611 3467
2534 2974
3317 2072
4281 2162
2780 4821
2443 4590
4240 1831
2867 2776
1745 2623
3084 2545
1686 3287
1476 4966
2035 4798
3890 4058
1547 2505
3905 1300
4447 4308
4960 1981
3721 3262
3456 2091
4228 3450
3032 3848
4456 2440
1737 2285
1767 4729
2074 1975
3980 4298
3639 3990
1861 2461
4631 2525
1615 4786
1806 2111
4497 2915
1243 3137
1410 3235
4391 2933
4143 1265
4233 1573
3405 3547
3533 3717
3688 1585
4590 2459
1299 1971
1727 1314
4473 4035
4020 1243
4402 3600
2609 4469
2326 1700
3751 4693
2147 1224
4631 4225
2227 4151
4162 3812
3278 4560
2727 1078
3006 4408
3960 3608
4548 3848
2113 2816
2153 3040
3110 3413
3660 2676
2614 1642
4778 4194
2695 1149
3902 4736
2230 1859
3798 4700
2649 3648
3498 2099
3219 1378
1243 2146
4832 1811
4703 1952
1232 4763
4765 1521
2723 3878
3779 4926
4892 3998
4906 4611
1084 3361
3836 2525
4723 4301
4891 3102
4458 1216
1491 3504
4779 4321
4083 1775
4142 1791
2257 3066
1187 3348
1126 1269
3817 4881
4331 4197
3028 1718
1489 1563
3768 2852
3697 4114
2353 3347
1967 3922
3958 1673
3335 3414
4491 3816
2032 3886
2643 3752
1142 4500
1325 3037
3662 1148
4938 1730
3724 3165
2273 1354
4439 2486
4967 4883
3352 4772
2755 1140
1200 4547
3928 2037
3905 2068
4404 1549
1177 2956
3967 4680
4004 1204
4347 2466
4211 2681
4064 3807
4506 2620
4350 1637
1288 2683
1499 1381
2764 2409
2320 3014
2726 3783
4632 2209
1617 1965
1406 4069
1481 4077
2064 2519
4852 3580
4502 2426
4586 1174
3181 3347
2903 3488
4624 4258
1958 3239
3285 2000
2823 4471
2940 4491
4322 1528
1138 2418
4494 2081
4859 1681
1187 1325
1464 2692
2228 1885
2413 1998
2110 3264
1851 1795
2450 1105
4515 1837
4660 4123
1263 1217
4687 1724
4504 2400
1196 2476
1517 3193
1101 4073
1395 4602
3005 1875
4473 2986
2419 3080
3837 2030
4613 4820
1962 4942
3612 3537
3722 1319
4997 3050
3796 4121
3261 1872
1759 3815
3568 2626
2092 2474
4986 4232
4079 4727
3861 2485
2367 1242
2940 4625
2781 1766
3067 1510
4764 3721
2777 3303
1588 2566
1271 3753
1092 4493
3402 2336
4552 1785
1407 4849
1210 3081
2729 3248
2984 1594
4774 2491
1393 4044
1370 2683
4971 1101
4449 4474
1031 1109
2856 2788
4056 2738
3042 4811
3401 3225
3156 3044
4252 4680
4569 3327
4358 1235
2923 3664
2409 3629
1447 4669
3822 4706
1650 2441
2557 4664
1349 1738
2101 2988
1036 1727
1340 3981
3400 4405
3979 2463
3752 4823
3341 2909
3564 3443
4785 2025
1426 2309
4271 1741
2668 3329
3245 4191
2648 2639
2457 4151
2131 2792
3448 3240
3136 1403
1962 3938
2209 1204
4900 4916
1443 2446
1383 2453
1062 3879
2162 4001
3771 4837
3506 1449
1167 2732
3064 3446
3307 2053
1491 1151
4408 1646
1855 2952
2110 2741
3121 4402
3531 2221
4646 2515
3074 2229
4791 4695
4244 3766
4887 2824
4291 3300
4179 3608
3783 2221
2417 4498
1505 3625
3263 4555
4122 1839
2652 4768
4928 1936
2727 1990
2340 3285
2985 3562
3188 4562
4832 1136
4046 2015
1411 1667
3914 1639
1861 2266
4463 4963
1792 3403
1643 2289
3999 1627
3999 4160
4441 4090
3360 1370
1270 4512
4517 2445
1696 3117
2413 1247
3160 3320
1483 4640
4504 2712
4026 3155
3423 3605
2214 2883
2028 4820
1332 2314
3398 3217
4579 2625
1302 2639
3158 1330
3546 2963
3685 3899
4548 1851
1502 4745
4338 2775
3541 4599
3112 1292
1678 2823
3313 3620
4143 3201
3210 4948
4742 4821
4142 1910
2052 3548
3620 2856
3323 3346
3303 4654
3637 2447
3602 4018
4332 1299
2308 4990
3634 3474
3886 2496
2533 1597
3936 1614
3800 1143
4942 2460
2214 4945
4140 4517
3531 1301
1897 2458
2304 3606
4737 1054
3848 2130
3223 2745
3810 4928
1667 4969
1029 3047
2586 4148
2244 1954
3113 1490
1777 4482
4277 1093
2668 3546
4025 1819
3550 1180
1329 1011
3107 4030
4707 1479
2359 2048
2253 2920
1365 3758
4691 4162
