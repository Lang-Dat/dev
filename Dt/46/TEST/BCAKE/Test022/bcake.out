10935882267059.359
